bootstrap/src/main/kotlin/jp/co/soramitsu/bootstrap/genesis/d3/D3TestGenesisFactory.kt

```diff
@@ -0,0 +1,112 @@
+package jp.co.soramitsu.bootstrap.genesis.d3
+
+import com.google.protobuf.util.JsonFormat
+import jp.co.soramitsu.iroha.java.Transaction
+import jp.co.soramitsu.iroha.java.TransactionBuilder
+import jp.co.soramitsu.iroha.testcontainers.detail.GenesisBlockBuilder
+import jp.co.soramitsu.bootstrap.genesis.*
+import java.util.*
+
+class D3TestGenesisFactory : jp.co.soramitsu.bootstrap.genesis.GenesisInterface {
+    override fun getAccountsNeeded(): List<jp.co.soramitsu.bootstrap.dto.AccountPrototype> = D3TestContext.d3neededAccounts
+
+    override fun getProject(): String = "D3"
+
+    override fun getEnvironment(): String {
```

Comment:
We can do the same as in function above.

Reply:
updated